src/Init/Data/BitVec/Bitblast.lean

```diff
@@ -290,7 +290,7 @@ theorem zeroExtend_truncate_succ_eq_zeroExtend_truncate_add_twoPow (x : BitVec w
         simp [hik', hik'']
   · ext k
     simp
-    omega
+    by_cases hi : x.getLsb i <;> simp [hi] <;> omega
```

Comment:
This feels like it could be shortened to:
```suggestion
    cases x.getLsb i <;> simp <;> omega
```

Reply:
Done.